packages/compass-aggregations/src/modules/index.ts

```diff
@@ -515,13 +367,7 @@ const MAPPINGS = {
   [RESTORE_PIPELINE]: doRestorePipeline,
```

Comment:
I'm planning to move these to the relevant parts of the app too, but keeping it out of this PR to not introduce too many changes at once